src/blocks/placeholders/index.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { registerBlockType } from '@wordpress/blocks';
+
+/**
+ * Placeholder blocks.
+ */
+const placeholders = [
+	{
+		name: 'site-title',
+		title: __( 'Site Title', 'woocommerce-gutenberg-emails' ),
+		description: __( 'Display the site title.', 'woocommerce-gutenberg-emails' ),
+	},
+	{
+		name: 'order-date',
+		title: __( 'Order Date', 'woocommerce-gutenberg-emails' ),
+		description: __( 'Display the date the order was placed.', 'woocommerce-gutenberg-emails' ),
+	},
+	{
+		name: 'order-number',
+		title: __( 'Order Number', 'woocommerce-gutenberg-emails' ),
+		description: __( 'Display the order number.', 'woocommerce-gutenberg-emails' ),
+	},
+	{
+		name: 'order-billing-full-name',
+		title: __( 'Billing Full Name', 'woocommerce-gutenberg-emails' ),
+		description: __( 'Display the full billing name from the order.', 'woocommerce-gutenberg-emails' ),
+	},
+	{
+		name: 'user-pass',
```

Comment:
Nitpicking here, but considering we are using `password` in all other block names, could we use it here too?
```suggestion
		name: 'user-password',
```

Reply:
I agree that the full name is better.  If we do this though, we break backwards compatibility if anyone copy/pastes an existing template that made use of `{user_pass}`.  Do you think the tradeoff here is worth it?

Reply:
Oh, I didn't think about that, good point. Let's leave it as `user-pass`, then.